backend/Lua/LuaLocalReference.cc

```diff
@@ -176,7 +202,11 @@ bool Local<Value>::isFunction() const {
   return val_ != 0 && lua_type(lua_backend::currentLua(), val_) == LUA_TFUNCTION;
 }
 
-bool Local<Value>::isArray() const { return isObject(); }
+bool Local<Value>::isArray() const { 
```

Comment:
Also, this check may be called really often, adding heavy logic may affect the performance.

Reply:
Ok,,, it may be the truth 😂
Thanks for your advice, anyway